PowerPointLabs/PowerPointLabs/PictureSlidesLab/Service/StylesDesigner.cs

```diff
@@ -41,7 +41,7 @@ public sealed class StylesDesigner : PowerPointPresentation, IStylesDesigner
         public StylesDesigner(Application app = null)
         {
             Path = TempPath.TempFolder;
-            Name = "PictureSlidesLabPreview" + Guid.NewGuid().ToString().Substring(0, 7);
```

Comment:
Not sure if this is the right answer. In some cases we store internal data as pptx files but we omit `.pptx` in the file name because we don't want users to open those files in PowerPoint and edit them.